src/main/java/uk/gov/ons/ssdc/caseprocessor/model/dto/SmsRequest.java

```diff
@@ -10,4 +11,6 @@
   private String phoneNumber;
 
   private String packCode;
+
+  private Map<String, String> uacMetadata;
```

Comment:
Should be `Object` or `String` to allow future flexibility beyond flat key:value pairs of strings